site/content/docs/user/ingress.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Ingress"
+menu:
+  main:
+    parent: "user"
+    identifier: "user-ingress"
+    weight: 3
+---
+
+## Setting Up An Ingress Controller
+
+We can leverage KIND's `extraPortMapping` config option when creating a cluster to
+forward ports from the host to an ingress controller running on a node. We can also specify 
+custom node label by using `node-labels` in the kubeadm `InitConfiguration`, to be used
+by the ingress controller `nodeSelector`.
+The following ingress controllers are known to work:
+
+ - [Ingress NGINX](#ingress-nginx)
+
+### Ingress NGINX
+
+Create a kind cluster with `extraPortMappings` and `node-labels`.
+
+```shell script
+cat <<EOF | kind create cluster --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    apiVersion: kubeadm.k8s.io/v1beta2
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+        authorization-mode: "AlwaysAllow"
```

Comment:
Wait a sec, what's this about? 😅

Reply:
i don't think this is needed for nginx ingress and it's not a recommended practice.

Reply:
I can confirm this is not required or recommended in ingress-nginx

Reply:
i suspect this is not needed.

Reply:
Yeah, if it's working without it we can probably safely remove this.